cdap-runtime-ext-dataproc/src/main/java/io/cdap/cdap/runtime/spi/provisioner/dataproc/DataprocProvisioner.java

```diff
@@ -98,18 +99,20 @@ public void validateProperties(Map<String, String> properties) {
     // Lower case letters and dashes allowed only.
     List<String> networkTags = conf.getNetworkTags();
     if (!networkTags.stream().allMatch(e -> NETWORK_TAGS_PATTERN.matcher(e).matches())) {
-      throw new IllegalArgumentException("Invalid Network Tags: Ensure tag length is max 63 chars"
-                                           + " and contains  lowercase letters, numbers and dashes only. ");
+      throw new DataprocRuntimeException("Invalid Network Tags: Ensure tag length is max 63 chars"
+        + " and contains  lowercase letters, numbers and dashes only. ", ErrorTag.CONFIGURATION);
     }
+
     if (networkTags.size() > 64) {
-      throw new IllegalArgumentException("Exceed Max number of tags. Only Max of 64 allowed. ");
+      throw new DataprocRuntimeException("Exceed Max number of tags. Only Max of 64 allowed. ",
+        ErrorTag.CONFIGURATION);
     }
 
     if (!isAutoscalingFieldsValid(conf, properties)) {
-      throw new IllegalArgumentException(
+      throw new DataprocRuntimeException(
         String.format("Invalid configs : %s, %s, %s. These are not allowed when %s is enabled ",
-                      DataprocConf.WORKER_NUM_NODES, DataprocConf.SECONDARY_WORKER_NUM_NODES,
```

Comment:
This looks like some custom indentation? I turned on the intellij setting "Use indentation relative to expression start", the indentation comes out the way it is below.

Reply:
that is the wrong setting, you should change it to be consistent with the rest of the project.